shuup/admin/template_helpers/shuup_admin.py

```diff
@@ -94,23 +94,24 @@ def get_breadcrumbs(context):
     return breadcrumbs
 
 
-def model_url(model, kind="detail", default=None):
+@contextfunction
+def model_url(context, model, kind="detail", default="#"):
```

Comment:
I think setting the default URL to # will make the UI appear broken to users. If we don't have the time to update stuff in templates (which is cool - I imagine that would take quite a bit more time), we should just leave things as-is and continue returning the "Can't view this page. You do not have the required permissions: set(['shuup.add_order'])". Maybe clean that permission string up by doing `','.join(permssions)`.